go/api/response_types.go

```diff
@@ -7,6 +7,11 @@ type Result[T any] struct {
 	isNil bool
 }
 
+type KeyWithMemberAndScore struct {
```

Comment:
Maybe we can call this BzpopBaseResponse and reuse it later for other BzPop commands?